client/blocks/comment-detail/comment-detail-header.jsx

```diff
@@ -82,7 +82,9 @@ export const CommentDetailHeader = ( {
 						</span>
 					</div>
 					<div className="comment-detail__author-info-element">
-						{ translate( 'on %(postTitle)s', { args: { postTitle } } ) }
+						{ translate( 'on %(postTitle)s', { args: {
+							postTitle: postTitle ? decodeEntities( postTitle ) : translate( 'Untitled' ),
```

Comment:
If there's no title, I believe it would be better for the user to take an excerpt from the post content. I think reader does this if you want to compare util functions.